src/domain/auth/authenticate.ts

```diff
@@ -11,9 +14,22 @@ import { TUNNISTAMO_API_TOKEN_ENDPOINT } from '../api/constants/ApiConstants';
 import { BackendTokenResponse } from './types/BackendAuthenticationTypes';
 
 export const loginTunnistamo = (path?: string) => {
-  userManager.signinRedirect(
-    path ? { data: { path: path } } : { data: { path: '/profile' } }
-  );
+  userManager
+    .signinRedirect(
+      path ? { data: { path: path } } : { data: { path: '/profile' } }
+    )
+    .catch(error => {
+      if (error.message === 'Network Error') {
```

Comment:
toast only 1 generic one, this is not needed

Reply:
It is not only the toast that is different, we're also not sending sentry errors for network problems